Documentation/ApiOverview/GlobalValues/Typo3ConfVars/Index.rst

```diff
@@ -112,20 +112,26 @@ may themselves be arrays.
 The configuration categories are:
 
 BE
-  Options related to the TYPO3 CMS backend
+  Options related to the TYPO3 CMS backend.
 
 DB
-  Database connection configuration
+  Database connection configuration.
+
+EXT
```

Comment:
Is EXT still of relevance? It used to contain the extension configuration in $GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf'] which has now been replaced by $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']. If there is more information about this already existing somewhere, might not be a bad idea to add it or add a link.

Reply:
Yes, there are some other settings under the EXT key:
[EXT][allowGlobalInstall]
[EXT][allowLocalInstall]
[EXT][excludeForPackaging]
are in the "Configure Installation-Wide Options" in v10 in the "Settings" module.